src/plugins/event_annotation/public/event_annotation_service/service.tsx

```diff
@@ -82,6 +82,7 @@ export function getEventAnnotationService(
     return {
       id: savedObject.id,
       references: savedObject.references,
+      namespaces: savedObject.namespaces ?? [],
```

Comment:
`namespaces` should always be present on the saved object, so we could also add an assertion operator rather than coercing to an empty array.